packages/block-library/src/buttons/content-justification-dropdown.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * WordPress dependencies
+ */
+import { DropdownMenu } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { justifyCenterIcon, justifyLeftIcon, justifyRightIcon } from './icons';
+
+const DEFAULT_ALLOWED_VALUES = [ 'left', 'center', 'right' ];
+
+const CONTROLS = {
+	left: {
+		icon: justifyLeftIcon,
+		title: __( 'Justify items left' ),
+	},
+	center: {
+		icon: justifyCenterIcon,
+		title: __( 'Justify items center' ),
+	},
+	right: {
+		icon: justifyRightIcon,
+		title: __( 'Justify items right' ),
+	},
+};
+
+const DEFAULT_ICON = CONTROLS.center.icon;
+
+/**
+ * Dropdown for selecting a content justification option.
+ *
+ * @param {Object}   props                 Component props.
+ * @param {string[]} [props.allowedValues] List of options to include. Default:
+ *                                         ['left', 'center', 'right'].
+ * @param {()=>void} props.onChange        Callback to run when an option is
+ *                                         selected in the dropdown.
+ * @param {Object}   props.toggleProps     Props to pass to the dropdown toggle.
+ * @param {string}   props.value           The current content justification
+ *                                         value.
+ *
+ * @return {WPComponent} The component.
+ */
+export default function ContentJustificationDropdown( {
```

Comment:
CSS actually has both [`justify-content`](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content) AND [`justify-items`](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items), and they do different things. Using the term "ItemsJustification" for what is actually content justification would be confusing, especially if we later add a control for the real item justification.

I don't think "inner block justification" is right either, since some blocks will have stuff that is affected by the control, but isn't an inner block.

Reply:
Ok, well it'd be appreciated if you could update the PR to use just the preferred term then:
https://github.com/WordPress/gutenberg/pull/23168/files#diff-087eec3915308be83b2b3833751ca31bR14-R26

Thanks.

Reply:
Done. :+1: